pennylane/devices/experimental/device_api.py

```diff
@@ -174,6 +196,34 @@ def preprocess(
         * validation of configuration parameters
         * choosing a best gradient method and ``grad_on_execution`` value.
 
+        .. details::
+            :title: Post processing function and derivatives
+
+            Derivatives and jacobian products will be bound to the machine learning library before the postprocessing
+            function is called on results. Therefore the machine learning library will be responsible for combining the
+            device provided derivatives and post processing derivatives.
+
+            .. code-block:: python
```

Comment:
code didn't render, i think you just need a new line here
```suggestion
            .. code-block:: python

```

Reply:
just added 🤞